shell/platform/darwin/macos/framework/Source/FlutterBackingStore.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+#import <Metal/Metal.h>
+
+/**
+ * A marker interface for backing store handles. Typically contain references to the buffers that
+ * are handed by the `FlutterView` to the `FlutterRenderer`.
+ */
+@interface FlutterRenderBackingStore : NSObject
+
+@end
+
+/**
+ * Wraps an OpenGL frame buffer.
+ */
+@interface FlutterOpenGLRenderBackingStore : FlutterRenderBackingStore
+
+/**
+ * Frame buffer ID referenced by this backing store instance.
+ */
+@property(nonatomic, readonly) uint32_t frameBufferID;
+
+/**
+ * Initializes a backing store with the specified frame buffer id.
+ */
+- (nullable instancetype)initWithFrameBufferID:(uint32_t)fboID;
```

Comment:
Nit: these could be nonnull; `NSObject`'s `init` is documented to never return `nil`.

Reply:
Good call, will make this nonnull